docs/eigenda/integrations-guides/dispersal/dispersal-api/overview.md

```diff
@@ -0,0 +1,166 @@
+
+:::warning
+It's important to note that during dispersal, blobs are automatically padded with zero bytes such that the final length of the blob in bytes is a multiple of 32. One implication of this is that a naive EigenDA client implementation will not retrieve the exact same blob that was dispersed.
+:::
+
+:::warning
+Every 32 bytes of data chunk is interpreted as an integer in big endian format where the lower address has more significant bits. The integer must stay in the valid range to be interpreted as a field element on the bn254 curve. The valid range is:
+
+```python
+0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617
+```
+
+...containing slightly less than 254 bits and more than 253 bits. If any one of the 32 bytes chunk is outside the range, the whole request is deemed as invalid, and rejected. For more detail see [Blob Serialization Requirements](../blob-serialization-requirements.md).
+:::
+
+<!-- TODO: Link to schema in code -->
+
+| __DisperseBlobRequest__ |
+| Field Name | Type | Description |
+| `data` | []byte | The data to be dispersed. The blob dispersed must conform to the [Blob Serialization Requirements](../blob-serialization-requirements.md) which ensure that the blob's KZG commitment may be representative of the original data that was sent to the disperser. |
+| `custom_quorum_numbers` | []uint32 | The quorums to which the blob will be sent, in addition to the required quorums which are configured on the EigenDA smart contract. If required quorums are included here, an error will be returned. The disperser will ensure that the encoded blobs for each quorum are all processed within the same batch. |
+| `account_id` | string | The account ID of the client. This should be a hex-encoded string of the ECSDA public key corresponding to the key used by the client to sign the BlobAuthHeader. |
+
+<!-- TODO: Follow up on whether this should just be an Ethereum address, not an ECDSA public key as mentioned in the docs. -->
+
+| __DisperseBlobReply__ |
+| Field Name | Type | Description |
+| `result` | BlobStatus | The status of the blob associated with the request_id. This field is returned in case the blob dispersal queuing fails immediately. If the blob was successfully dispersed, this field will be set to `PROCESSING` (`1`). |
+| `request_id` | []byte | The request ID generated by the disperser corresponding to the dispersal. Once a request is accepted (although not processed), a unique request ID will be generated. Two different DisperseBlobRequests (determined by the hash of the DisperseBlobRequest) will have different IDs, and the same DisperseBlobRequest sent repeatedly at different times will also have different IDs. The client should use this ID to query the processing status of the request (via the `GetBlobStatus()` API). |
+
+#### Authentication
```

Comment:
Some comments:
- Let's not refer to IP-based whitelisting as authentication. Probably not much needs to be said about this here. 
- I'm confused about how all of this relates to the contents of the `ecdsa-keypair-authentication` page. 

Suggested separation: 
The API overview describes the behavior of the API. E.g., it expects to receive this message (which must be signed in such a way), returns that message, etc.  This page should also indicate the requirements for the private key that is used (i.e., only a signature is required, we recommend no funds at this account). 

The how-to-guide should give a practical walk through of how to use the client. Can contain bash code for generating the new wallet, etc. 

Both pages should make reference to each other.

Reply:
I'll update this section here, and move the keypair generation doc to the client how-to PR which is in flight now.